### ISSUE TITLE
Update of Mod profile

### ISSUE THREAD

**Issue by VinciBeaulo** [Ethanol10/darth-vader-ror2#2]
I hope you can improve the configuration file of this mod of Darth Vader. I hope you can adjust some characters' attributes, such as energy or cooling time. I want to adjust some balance myself to make this mod more in line with my idea of Darth Vader.

**Comment by Ethanol10** [comment]
What sort of stats would you like to see have configs?

**Comment by VinciBeaulo** [comment]
For example, some basic data of Darth Vader in two passive forms, the basic value of energy recovery and the increase after upgrading, the cooling time of each skill, the reduced cooling time after attack and the value bonus after upgrading; The basic value of Vader itself, the initial damage of the skill and the bonus after upgrading.

**Comment by Ethanol10** [comment]
Sorry, what do you mean by upgrading? Do you mean using R( rage) to get a temporary stay boost or getting items to upgrade your stats?

**Comment by VinciBeaulo** [comment]
In the risk of rain game, the attribute bonus brought by the level increase of playing monsters is then the adjustment of skill damage in the configuration file and the adjustment of skill cooling in another form. So I want you to update some adjustments in the configuration file so that I can balance Darth Vader myself.

**Comment by Ethanol10** [comment]
So, from what I get you want the ability to modify the stats below:


- Primary Coefficient / Cooldown (Cooldown system)
- Secondary Coefficient / Cooldown (Cooldown system)
- Utility multiplier / Cooldown (Cooldown system)
- Special Duration / Cooldown (Cooldown system)
- Energy Recovery (Energy System)
- Passive energy recovery (Energy System)
- Energy gained on hit (Energy System)

**Comment by VinciBeaulo** [comment]
Yes, there is also the hope that the basic value of the character can be adjusted, such as ordinary attack and skill. Of course, it is also better to adjust the growth rate of the upgraded skill. Because in many cases, Vader only uses ordinary attack to cause low damage, and the effect that ordinary attack cannot trigger some props, and the secondary skill is useless for the boss, I hope it can become energy consumption, drag or directly cause damage.

**Comment by Ethanol10** [comment]
The pull/push is nerfed on bosses because it does too much to displace the enemy, especially when you have infinite uses on rage. We can modify the base damage.

**Comment by VinciBeaulo** [comment]
I think this is also very good. Thank you for replying to my feedback, and look forward to your next update!
